std_msgs/MultiArrayDimension dim
Object1 object
string ichbinString
int64 ichbinZahl
